Gemfile

```diff
@@ -80,6 +82,7 @@ group :development, :test do
   gem "debug", platforms: %i[mri mingw x64_mingw]
   gem "dotenv-rails", "~> 2.0"
   gem "erb_lint", require: false
+  gem "ostruct"
```

Comment:
I thought this was already removed - maybe just deprecated before.
Anyway, great!